haskell/sorting-in-reverse-order.md

```diff
@@ -0,0 +1,24 @@
+# Sorting in reverse order
+
+`Data.Ord` contains a newtype, `Down`, which can wrap any type that has an
+`Ord` instance, in order to provide a reversed `Ord` instance for that type.
+
+For example:
+
+```hs
```

Comment:
Or, actually, this isn't valid haskell code, since it has a prelude prompt. How about leaving off the language here and below?